test/library/characterization/test_qubit_spectroscopy.py

```diff
@@ -208,6 +211,10 @@ def test_kerneled_expdata_serialization(self):
         exp = QubitSpectroscopy([qubit], frequencies)
 
         exp.set_run_options(meas_level=MeasLevel.KERNELED, shots=1024)
+
+        # Checking serialization of the experiment obj
+        self.assertRoundTripSerializable(exp)
```

Comment:
Why do we also need the check before `exp.run`?

Reply:
I wanted to do it for the completeness of the test. I will delete it as this is not what this PR meant to fix.